### PR TITLE
Clarifications

### PR DESCRIPTION
**Pull request by andynava**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/architecture/cloud-native/leverage-serverless-functions.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ ms.date: 06/30/2019
 
 [!INCLUDE [book-preview](../../../includes/book-preview.md)]
 
-In the spectrum of managing full machines and operating systems to leveraging cloud capabilities, serverless lives at the extreme end where the only thing you're responsible for is your code, and you only pay for your code when it runs. Azure Functions provides a way to build serverless capabilities into your applications.
```
Comment (andresnava) [inline docs/architecture/cloud-native/leverage-serverless-functions.md]: You don't pay for your code.

File: docs/architecture/cloud-native/leverage-serverless-functions.md [hunk 2]
```diff
@@ -7,13 +7,13 @@ ms.date: 06/30/2019
...
+In the spectrum of managing full machines and operating systems to leveraging cloud capabilities, serverless lives at the extreme end where the only thing you're responsible for is your code, and you only pay when your code runs. Azure Functions provides a way to build serverless capabilities into your applications. 
 
 ## What is serverless?
 
 Serverless computing doesn't mean there isn't a server involved in running your application - the code still runs on a server somewhere. The distinction is that the application development team no longer needs to concern themselves with managing server infrastructure. Serverless computing solutions like Azure Functions help teams increase their productivity and allow organizations to optimize their resources and focus on delivering solutions.
 
-Serverless computing uses event-triggered stateless containers to host your application or part of your application. Serverless platforms can scale up and down to meet demand as-needed. Platforms like Azure Functions have easy direct access to other Azure services like queues, events, and storage.
```
Comment (andresnava) [inline docs/architecture/cloud-native/leverage-serverless-functions.md]: Serveless platforms scale out and in, but not up and down.

File: docs/architecture/cloud-native/leverage-serverless-functions.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ Many applications have long-running processes as part of their workflows. Often
 
 ## When should you avoid serverless?
 
-Serverless computing is best-used for tasks that don't block the user interface. This means they're not ideal for hosting web applications or web APIs directly. The main reason for this is that serverless solutions are provisioned and scaled on demand. When a new instance of a function is needed, referred to as a *cold start*, it takes time to provision. This time is typically a few seconds, but can be longer depending on a variety of factors. A single instance can often be maintained indefinitely (for instance, by periodically making a request to it), but the cold start issue remains if the number of instances ever needs to scale up.
```
Comment (andresnava) [inline docs/architecture/cloud-native/leverage-serverless-functions.md]: "maintained indefinitely" -> "maintained active indefinitely". Maybe "maintained alive indefinitely"?

**Comment by mairaw** [comment]
@ardalis @nishanil please review

**Comment by mairaw** [comment]
Tagging @robvet too.

**Comment by robvet** [comment]
Let Steve drive this one, but think we might keep some of the original description and add some of the suggested.

File: docs/architecture/cloud-native/leverage-serverless-functions.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ Many applications have long-running processes as part of their workflows. Often
...
+Serverless computing is best-used for tasks that don't block the user interface. This means they're not ideal for hosting web applications or web APIs directly. The main reason for this is that serverless solutions are provisioned and scaled on demand. When a new instance of a function is needed, referred to as a *cold start*, it takes time to provision. This time is typically a few seconds, but can be longer depending on a variety of factors. A single instance can often be maintained active indefinitely (for instance, by periodically making a request to it), but the cold start issue remains if the number of instances ever needs to scale up.
```
Comment (mvelosop) [inline docs/architecture/cloud-native/leverage-serverless-functions.md]: "kept alive" perhaps?
```suggestion
Serverless computing is best-used for tasks that don't block the user interface. This means they're not ideal for hosting web applications or web APIs directly. The main reason for this is that serverless solutions are provisioned and scaled on demand. When a new instance of a function is needed, referred to as a *cold start*, it takes time to provision. This time is typically a few seconds, but can be longer depending on a variety of factors. A single instance can often be kept alive indefinitely (for instance, by periodically making a request to it), but the cold start issue remains if the number of instances ever needs to scale up.
```

Comment (ardalis) [reply docs/architecture/cloud-native/leverage-serverless-functions.md]: "kept alive" works.

**Comment by ardalis** [comment]
Swap in "kept alive" and then ship it.

**Review by mvelosop — APPROVED** [review]
Great, so this is ready to ship! 😊

**Pull request merged**